packages/utils/src/create-mr-for-patients.ts

```diff
@@ -1,14 +1,20 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
 import fs from "fs";
 import https from "https";
 import { MetriportMedicalApi } from "@metriport/api-sdk";
-import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import { getEnvVar, getEnvVarOrFail } from "@metriport/core/util/env-var";
 import { Sequelize } from "sequelize";
 import { sleep } from "@metriport/core/util/sleep";
 
 const apiUrl = getEnvVarOrFail("API_URL");
 const apiKey = getEnvVarOrFail("API_KEY");
 const cxId = getEnvVarOrFail("CX_ID");
 const sqlDBCreds = getEnvVarOrFail("DB_CREDS");
+const orgName = getEnvVar("ORG_NAME");
+
+const DIR_NAME = `${orgName}_MR_Summaries`;
```

Comment:
NABD, and I know this has been merged, but if you're still working on it, let's please use dash `-` to split customer name (`orgName.replaceAll(" ", "-")`) and underscore `_` to separate different components of the file name, like [this one](https://github.com/metriport/metriport/blob/6ccba4c0e0693ac3a45b22dacb4ae21ba24a3d12/packages/utils/src/commonwell/coverage-assessment.ts#L52). Just so files get ordered nicely after multiple runs. :)